### ISSUE TITLE
Favorite Heart Icon on Individual BB Page Should Be Solid when Active

### ISSUE THREAD

**Issue by tdhartwick** [zurb/building-blocks#59]
When the user favorites on an individual BB page, the Icon should be filled solid blue.

**Comment by abdullahsalem** [comment]
@tdhartwick You're right. But I think the current SVG cannot be filled as we expected because the Compound-Paths were drawn as the following:
![like-icon](https://cloud.githubusercontent.com/assets/177020/25063013/a5559aa6-21e1-11e7-9eba-aa330b6b0272.png)
So when we fill that path `fill: blue`, we actually fill that a little inner pace -as shown above- not the whole inner heart. Therefore, I think we need to draw the heart itself in a way that allows us to fill the whole inner space.
